app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use App\Category;
+
+class CategoryController extends Controller
+{
+    public function open_category_admin()
+    {
+        $categories = Category::get();
+        // dd(count($user));
+        if (count($categories) == 0) {
+            return redirect(route('add_category_admin'));
```

Comment:
This should be `redirect()->route('add_category_admin')`. You can read from here: https://laravel.com/docs/5.7/redirects#redirecting-named-routes